src/quo2/components/navigation/bottom_nav_tab.cljs

```diff
@@ -1,71 +1,77 @@
 (ns quo2.components.navigation.bottom-nav-tab
   (:require [quo.react-native :as rn]
-            [reagent.core :as reagent]
+            [quo2.reanimated :as reanimated]
             [quo2.foundations.colors :as colors]
-            [quo2.components.icon :as icon]
+            [quo2.components.icons.icons :as icons]
             [quo2.components.counter.counter :as counter]))
 
 (defn toggle-background-color [background-color press-out? pass-through?]
-  (let [color (cond
-                press-out?    nil
-                pass-through? colors/white-opa-5
-                :else         colors/neutral-70)]
-    (reset! background-color color)))
+  (reanimated/set-shared-value
+   background-color
+   (cond
+     press-out?    "transparent"
```

Comment:
Do we use transparent as a string or a keyword ?

Reply:
Hi @ibrkhalil, Thank you very much for PR review.
Actually I tried using :transparent keyword, but it is not working for ios